nixos/tests/systemd-initrd-luks-ultrablue.nix

```diff
@@ -0,0 +1,55 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }: let
+
+  keyfile = pkgs.writeText "luks-keyfile" ''
+    MIGHAoGBAJ4rGTSo/ldyjQypd0kuS7k2OSsmQYzMH6TNj3nQ/vIUjDn7fqa3slt2
+    gV6EK3TmTbGc4tzC1v4SWx2m+2Bjdtn4Fs4wiBwn1lbRdC6i5ZYCqasTWIntWn+6
+    FllUkMD5oqjOR/YcboxG8Z3B5sJuvTP9llsF+gnuveWih9dpbBr7AgEC
+  '';
+
+in {
+  name = "systemd-initrd-luks-keyfile-with-ultrablue";
+
+  # This tests that ultrablue does not break boot if bluetooth is unavailable.
+  nodes.machine = { pkgs, ... }: {
+    # Use systemd-boot
+    virtualisation = {
+      emptyDiskImages = [ 512 ];
+      useBootLoader = true;
+      useEFIBoot = true;
+    };
+    boot.loader.systemd-boot.enable = true;
+
+    environment.systemPackages = with pkgs; [ cryptsetup ];
+    boot.initrd.systemd = {
+      enable = true;
+      emergencyAccess = true;
+      ultrablue.enable = true;
```

Comment:
What is ultrablue actually doing in this test? As far as I can tell, this test just creates an encrypted drive and reboots with it as the `bootDevice`.

Reply:
I want to ensure that Ultrablue being in the chain **does not prevent** normal booting with a key, i.e. breaking LUKS setups.